src/utils/ReviewsStats.js

```diff
@@ -19,14 +19,20 @@ export class ReviewsStats {
       const project = this.projects[id];
       // Set the average turnaround time for each project
       project.avgTurnaroundTime = project.totalTurnaroundTime / project.totalAssigned;
-      // Find the average daily earnings
+    });
+
+    // Find the average daily earnings
+
+    // Checks if the endTime is today or in the future. If yes it calculates
+    // the number of days considering the endDate as the current time. If not,
+    // it uses the endTime provided by the user.
+    const endIsToday = moment.utc().diff(this.endDate, 'days', true) < 1;
```

Comment:
There is the [isSame()](http://momentjs.com/docs/#/query/is-same/) method in moment. Since the code is being updated I thought it might be a more descriptive way of writing it.